src/mmw/apps/modeling/tasks.py

```diff
@@ -201,10 +201,14 @@ def add_huc12(srat_huc12, aggregate):
 
 
 @shared_task(throws=Exception)
-def nlcd_soil(result):
-    if 'error' in result:
-        raise Exception(f'[nlcd_soil] {result["error"]}')
+def nlcd_soil_tr55(results):
+    if 'error' in results:
+        raise Exception(f'[nlcd_soil_tr55] {results["error"]}')
+
+    return [nlcd_soil(result) for result in results]
```

Comment:
This is just a simple wrapper around the old one, which runs it for every `result` in the output of RasterGroupedCountMany.